eos/constraints/B-to-V-form-factors.yaml

```diff
@@ -1,5 +1,20 @@
 # vim: set fdm=marker :
 
+#########################
+# B -> D^* form factors #
+#########################
+# {{{
+B->D^*::A_1[s_max]@HDW-2017:
```

Comment:
This is from the paper by the HPQCD collaboration, correct? I think it's a better idea
to name it HPQCD-2017.